spec/services/sync_all_providers_from_find_spec.rb

```diff
@@ -19,6 +19,21 @@
       expect { SyncAllProvidersFromFind.call }.to change { Provider.count }.by(2)
     end
 
+    it 'sets the last updated timestamp' do
```

Comment:
another test for "it doesn't set the last updated timestamp" in case of exception? useful?